src/MusicStore/Components/AnnouncementComponent.cs

```diff
@@ -10,18 +10,23 @@ namespace MusicStore.Components
     [ViewComponent(Name = "Announcement")]
     public class AnnouncementComponent : ViewComponent
     {
-        private readonly MusicStoreContext _dbContext;
-        private readonly IMemoryCache _cache;
+        [Activate]
+        public MusicStoreContext DbContext
+        {
+            get;
+            set;
+        }
 
-        public AnnouncementComponent(MusicStoreContext dbContext, IMemoryCache memoryCache)
+        [Activate]
+        public IMemoryCache MemoryCache
```

Comment:
@Praburaj Well. maybe we can use Cache just like using dbContext for MusicStoreContext?? I can change them to Cache if @pranavkm agree.